Rogue-Ape-Studios-XR/Secrets-of-Ignacios
# ✨ Alternative casting modes

## Content

Adds a timer option, so the spell shoots after a set amount of time (can be changed in the inspector). also a new gesture has been added to take the place of the claw if people prefer it.

## Changes

### Added
`Assets/Samples/XR Hands/1.4.1/Gestures/Examples/Hand Poses/Fist Forward.asset` : Was created. New gesture.

### Updated
`Assets/Scripts/Spells/SpellManager.cs` : Was updated / renamed. Changed to have a timer feature.
`Assets/Samples/XR Hands/1.4.1/Gestures/HandGestures.unity` : Was updated / renamed. New gesture can be tested here.
`Assets/Prefabs/PlayerRig/MR Interaction Setup.prefab` : Was updated / renamed. Added the new casting options and references have been dragged in.


## Testing

The feature / Bugfix can be testing by following these steps:

1. Place player prefab in a scene.
2. Check timer cast option for timer to be activated.
3. Use new gesture to show indicator or use normal cast gesture with the timer.
